lwb: 0.265 
Tf: 1.5
N: 30
Qc: 100
Ql: 1
Q_theta: 100
R_d: 0.01
R_delta: 0.01
nTimes: 100
r: 0.01
d_min: -3.0
d_max: 5.0
ddot_min: -10.0
ddot_max: 10.0
delta_min: -50
delta_max: 50 
deltadot_min: -2  
deltadot_max: 2 
thetadot_min: -0.1  
thetadot_max: 5 
theta_min: 0.00  
theta_max: 100 
vx_min: 0 
vx_max: 0.71
loop_rate: 20
